swiftsimio/optional_packages.py

```diff
@@ -7,6 +7,7 @@
 + scipy.spatial: KDTrees
 + sphviewer: visualisation
 """
+from numba.cuda.cudadrv.error import CudaSupportError
```

Comment:
Hmm this is quite tricky; the user may not have numba installed at all (hence it itself is an optional package). Can we mock this up?

Reply:
When I check for cuda, I am catching an additional exception now.